code/__DEFINES/traits.dm

```diff
@@ -359,6 +359,10 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_DONT_WRITE_MEMORY "dont_write_memory"
 /// This mob can be painted with the spraycan
 #define TRAIT_SPRAY_PAINTABLE "spray_paintable"
+/// This person is blushing
+#define TRAIT_BLUSHING "blushing"
+/// This person is blushing due to being drunk
+#define TRAIT_DRUNK_BLUSHING "drunk_blushing"
```

Comment:
Traits can actually be assigned and handled from different sources, meaning you don't need both of these.
You only need one trait: `TRAIT_BLUSHING`

Reply:
Thanks for the review! I've added EMOTE_TRAIT and DRUNK_TRAIT defines to use as trait sources